inst/BS5/templates/content-title-body.html

```diff
@@ -11,7 +11,7 @@ <h1>{{pagetitle}}</h1>
   </main>
 
   <aside class="col-md-3">
-    <nav id="toc">
+    <nav id="toc" aria-label="Table of contents">
```

Comment:
Should this actually use a translated string?

Reply:
Oh yeah, good point. Hopefully there's an existing string we can use.